tests/idl/generate.sh

```diff
@@ -15,4 +15,10 @@ cd ../generics
 anchor idl build -o $dir/generics_build.json
 
 cd ../relations-derivation
+IDL_PARSE=$(anchor idl parse --file src/lib.rs -o $dir/relations_build_parse.json)
+if echo "${IDL_PARSE}" | grep -q 'WARNING:'; then
+  echo "IDL parse has warnings:"
+  echo "${IDL_PARSE}"
+  exit 1
+fi
```

Comment:
Thanks but this currently only tests whether there is a `WARNING:`. It would be better to test relations derivation in `tests` directory:

```ts
it("Can use relations derivation with seed constant", async () => {
  await program.methods.testSeedConstant().accounts({}).rpc();
});
```

If seed constant works as expected, the test above should pass.

Reply:
Sorry, I completely missed those tests. I've just updated it with your recommendations, thank you.

Reply:
Sorry, I've completely missed those tests. I've just updated it with your recommendations, thank you.